pyvlx/pyvlx.py

```diff
@@ -39,42 +35,35 @@ def __init__(self, path=None, host=None, password=None, loop=None):
         self.scenes = Scenes(self)
         self.version = None
         self.protocol_version = None
+        self.klf200 = Klf200Gateway(pyvlx=self)
 
     async def connect(self):
         """Connect to KLF 200."""
-        PYVLXLOG.debug("Connecting to KLF 200.")
+        PYVLXLOG.warning("Connecting to KLF 200.")
```

Comment:
I would appreciate to keep this as a debug message.

Reply:
set to debug